### ISSUE TITLE
add button to UI so user can toggle thumbnail drawer to show/hide

### ISSUE THREAD

**Issue by surreal8** [ProjectMirador/mirador#2884]
In Mirador 2 we have the option to show/hide the thumbnail 'drawer' via the 3 dots on top of the UI element. This functionality is missing from Mirador 3. It would be great if a UI element could be added on the thumbnail drawer so the user can show/hide the drawer when desired.

**Comment by ggeisler** [comment]
@surreal8 If I understand you correctly, we didn't actually remove this functionality in Mirador 3. We moved it:

<img width="719" alt="Screen Shot 2019-10-07 at 4 08 28 PM" src="https://user-images.githubusercontent.com/101482/66355660-e8774f00-e91c-11e9-821b-64021645da2f.png">

**Comment by surreal8** [comment]
@ggeisler yes, I saw that but didn't consider that to be the same as the toggle in Mirador 2. In Mirador 2, the thumbnails are 'on' but you can use toggle to hide/show. In Mirador 3, it seems this has been rethought a little differently. I guess for our implementation I would like a UI element to toggle the thumbnail display since we are hiding everything else in the UI... 
![Mirador_Viewer2](https://user-images.githubusercontent.com/925523/66415395-1ada8800-e9c1-11e9-9d4a-3977b601a0f6.jpg)

**Comment by ggeisler** [comment]
@surreal8 Yeah, I see what you mean. In Mirador 3 we added the option of showing the thumbnails on the right as well as bottom and for other reasons (I believe having to do with not wanting to add to the visual clutter in the thumbnail area because in M3 we're also showing the page counter labels and previous | next buttons) decided to move the control to the menu.

But I see the predicament when you configure all the UI controls off in Mirador 3; the user either gets thumbnails always or never.

This probably needs some design thought (e.g., would it make sense to add an affordance similar to Mirador 2, but have it off by default and hopefully only adopters doing a minimal configuration like you would turn it on?). We're not actively working on Mirador at the moment so not sure when we might have time to consider this more. If you have ideas or proposals, definitely add them here.

**Comment by hrvoj3e** [comment]
Is there an config option (window or windows) in Mirador 3 to hide thumbnails (hiding it completely from UI)?
We want to hide it when we are showing only one image and thumbnails bar "gets in the way" visually.

**Comment by marlo-longley** [comment]
@hrvoj3e this is a long overdue reply, but yes:
```
thumbnailNavigation: {
  defaultPosition: 'off'
}
```
'off' is actually already the global default. To set it per-window:
```
windows: [{
  thumbnailNavigationPosition: 'off'
}]
```

There is currently no way to conditionally set this in the case of a single image. But in my mind this relates to a different idea of having more reasonable defaults for single canvases.

**Comment by marlo-longley** [comment]
Audit 2026: this seems like an issue of setting discoverability. The old M2 toggle seemed more handy.
